widgets/common/src/deployModal/GenericDeployModal.tsx

```diff
@@ -143,11 +152,18 @@ type GenericDeployModalState = {
     loading: boolean;
     loadingMessage: string;
     runtimeOnlyEvaluation: boolean;
-    showInstallModal: boolean;
     siteName: string;
     skipPluginsValidation: boolean;
     visibility: any;
     workflow: Workflow;
+    baseWorkflowParams: BaseWorkflowInputs;
+    userWorkflowParams: UserWorkflowInputsState;
```

Comment:
I'm wondering shouldn't we change the names to indicate more precisely what kind of workflow we are using here:
```suggestion
    installWorkflow: Workflow;
    baseInstallWorkflowParams: BaseWorkflowInputs;
    userInstallWorkflowParams: UserWorkflowInputsState;
```

In the context of `GenericDeployModal` there cannot be any other workflow, am I correct?

Reply:
I added `install` prefix: [daa561f](https://github.com/cloudify-cosmo/cloudify-stage/pull/1812/commits/daa561f4a97f0ae1e757180a6b800d71b8cd9212)